Dockerfile

```
@@ -1,4 +1,4 @@
-FROM terascope/node-base:14.17.0
+FROM terascope/node-base:14.17.5
```

I think there is a `14.17.6` now.

```suggestion
FROM terascope/node-base:14.17.6
```